test/java/clojure/lang/UtilTest.java

```diff
@@ -0,0 +1,227 @@
+package clojure.lang;
+
+import com.sun.org.apache.xpath.internal.operations.String;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import java.math.BigInteger;
+import java.util.HashMap;
+
+public class UtilTest {
+
+  @Rule public final ExpectedException thrown = ExpectedException.none();
+
+  @Test
+  public void testClassOf() {
+    Assert.assertNull(Util.classOf(null));
+    Assert.assertEquals(Util.classOf(0), Integer.class);
+  }
+
+  @Test
+  public void testCompare() {
+    Assert.assertEquals(1, Util.compare(false, null));
+    Assert.assertEquals(0, Util.compare(1, 1));
+    Assert.assertEquals(-1, Util.compare(null, 0));
+    Assert.assertEquals(
+            0, Util.compare(new Integer(1), new Integer(1)));
+  }
+
+  @Test
+  public void testEquals() {
+    Assert.assertTrue(Util.equals(1, 1));
+    Assert.assertFalse(Util.equals(1, 0));
```

Comment:
Add a blank between different types of Asserts.

Reply:
Agree, this complies with Assertion point 4 in the checklist style guide 👍